apollo-router/src/query_planner/tests.rs

```diff
@@ -420,7 +421,12 @@ async fn defer_if_condition() {
           }"#;
 
     let schema = include_str!("testdata/defer_clause.graphql");
-    let schema = Arc::new(Schema::parse_test(schema, &Default::default()).unwrap());
+    // we need to use the planner here instead of Schema::parse_test because that one uses the router bridge's api_schema function
+    // does not keep the defer directive definition
```

Comment:
Is this still relevant after https://github.com/apollographql/router/pull/4701 ?

Reply:
Yeah it should be possible to go back to using `parse_test` here as it uses rust api schema now